server/data/restClient.test.ts

```diff
@@ -11,7 +11,14 @@ describe('restClient', () => {
 
   beforeEach(() => {
     config.apis.approvedPremises.url = 'http://localhost:8080'
-    fakeApprovedPremisesApi = nock(config.apis.approvedPremises.url)
+    config.apis.approvedPremises.serviceName = 'approved-premises'
+
+    fakeApprovedPremisesApi = nock(config.apis.approvedPremises.url, {
```

Comment:
Is `nock` doing some magic here so that if the bearer token is not set, it'll go bang and let us know? That'd make me confident in the tidying up we do to lines 68 and 82.

Reply:
Yep - the new `reqheaders` config at line 17